distribution/docs/src/main/resources/content/_integrating/_endpoints/other-endpoints-intro.adoc

```diff
@@ -0,0 +1,14 @@
+:type: endpointIntro
+:status: published
+:title: Other Endpoints
+:operations: other
+:order: 06
+
+== {title}
+
+${branding} also includes several specialized endpoints for specific purposes.
```

Comment:
❓ Should we remove the word "several"? There's only 2 in DDF.

Reply:
yes. removed